File: Лабораторная 2/WindowsFormsApp5/ITransport.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WindowsFormsApp5
{
    public interface ITransport
    {
        void moveShip(Graphics g);
        void drawShip(Graphics g);
        void setPosition(int x, int y);
        void loadPassenger(int count);
        int getPassenger();
        void SetMainColor(Color color);
        string getInfo();
    }
}

File: lab_2_TP/src/lab_2_TP/ITransport.java
package lab_2_TP;

import java.awt.Color;
import java.awt.Graphics;
import java.io.Serializable;
public interface ITransport extends Serializable {
	void moveShip(Graphics g);

	void drawShip(Graphics g);

	void setPosition(int x, int y);

	void loadPassenger(int count);

	int getPassenger();
	
	void setMainColor(Color color);

	String getInfo();
	
	

	
}